cli/ceph/ceph.py

```diff
@@ -84,3 +84,8 @@ def osd(self, command, **kw):
         cmd = f"{self.base_cmd} osd {command}"
         cmd += build_cmd_from_args(**kw)
         return self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
+
+    def ceph_config(self, command, **kw):
```

Comment:
Move this method to [1] accordingly
https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/config.py